python/tvm/relay/frontend/tflite.py

```diff
@@ -1488,6 +1511,42 @@ def convert_split(self, op):
 
         return out
 
+    def convert_split_v(self, op):
+        """SPLIT_V implementation."""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+
+        assert len(input_tensors) == 3, "input tensors length should be == 3"
```

Comment:
rephrase to -> input tensors length should be 3

Reply:
Done